bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js

```diff
@@ -1,9 +1,7 @@
 module.exports = (function () { // eslint-disable-line no-unused-vars
-  var _ = require('lodash');
-  var assign = _.assign;
-  var defaults = _.defaults;
-  var isUndefined = _.isUndefined;
-  var trim = _.trim;
+  function isUndefined (suspect) {
+    return suspect === undefined; // eslint-disable-line no-undefined
```

Comment:
We'd have a real broken site if it were the case, but it avoids the edge case where `undefined` is reassigned. No reason to write more bulletproof code here when it's only a few more characters.

Lodash actually protects for this edge case in a different way here: https://github.com/lodash/lodash/blob/4.15.0/lodash.js#L11-L12